the-experts-BE/app.js

```diff
@@ -4,38 +4,35 @@ const path = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 
-const indexRouter = require("./routes/index");
-const usersRouter = require("./routes/users");
-const testAPIRouter = require("./routes/testAPI");
 const getContractor = require("./routes/contractor-results");
 const cors = require("cors");
 
 const app = express();
 
 // view engine setup
-app.set("views", path.join(__dirname, "views"));
-app.set("view engine", "jade");
+// app.set("views", path.join(__dirname, "views"));
+// app.set("view engine", "jade");
 app.use(cors());
 
-app.use(logger("dev"));
+// app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 
-app.use("/users", usersRouter);
-app.use("/testAPI", testAPIRouter);
 app.use("/contractor-results/:job", getContractor);
 
-if (process.env.NODE_ENV === 'production') {
-
-  app.use(express.static(path.join(__dirname, '..', 'the-experts-fe', 'build')));
-  app.get('*', (req, res) => {
-    res.sendFile(path.join(__dirname,'..', 'the-experts-fe', 'build', 'index.html'));
+if (process.env.NODE_ENV === "production") {
+  app.use(
+    express.static(path.join(__dirname, "..", "the-experts-fe", "build"))
+  );
+  app.get("*", (req, res) => {
+    res.sendFile(
+      path.join(__dirname, "..", "the-experts-fe", "build", "index.html")
+    );
   });
 }
 
-
 // catch 404 and forward to error handler
 app.use(function(req, res, next) {
```

Comment:
This function can actually be deleted now. Think about it, you got an `app.get("*", ...)` route which will match anything that's not already been matched by your other routes, and it will return your `index.html`. Even if I did `https://the-experts.herokuapp.com/sdfjsdjflsfjs`, it would still serve up your home page. Now you don't need to worry about 404 errors.